src/utils/api.test.ts

```diff
@@ -11,6 +11,22 @@ describe('api', () => {
   };
 
   describe('handleResponseType', () => {
+    test('should handle missing Content-Type', async () => {
+      const response: Response = {
+        url: 'http://localhost:8080/-/packages',
+        ok: false,
+        // @ts-ignore
+        headers: {
+          get: () => null,
+        } as Headers,
+      } as Response;
+
+      const handled = await handleResponseType(response);
+
+      // Should this actually return [false, null] ?
```

Comment:
According the `fetch` https://developer.mozilla.org/en-US/docs/Web/API/Response#Properties docs the `Response ` . The first part of the array should be boolean and thus as second argument a the promise resolved according the type `blob` etc etc ...